### PR TITLE
Update README.adoc

### PR DESCRIPTION
**Pull request by smccarthy-ie**
Update alternative product name for RHOSAK and link to Streams for Apache Kafka docs

### CODE REVIEW

**Comment by smccarthy-ie** [comment]
Hey @PaulRMellor, Very minor one for approval, Just updating the Streams product name as alternative product for RHOSAK :) Thanks!

File: README.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@
 ====
 The cloud services described on this page are no longer available. For self-managed alternatives, see the following products:
 
-* OpenShift Streams for Apache Kakfa: see link:https://access.redhat.com/documentation/en-us/red_hat_amq_streams[Red Hat AMQ Streams]
+* OpenShift Streams for Apache Kakfa: see link:https://docs.redhat.com/en/documentation/red_hat_streams_for_apache_kafka/[Red Hat Streams for Apache Kakfa]
```
Comment (PaulRMellor) [inline README.adoc:14]:
```suggestion
* OpenShift Streams for Apache Kakfa: see link:https://docs.redhat.com/en/documentation/red_hat_streams_for_apache_kafka/[Streams for Apache Kakfa]
```

**Comment by PaulRMellor** [comment]
> Hey @PaulRMellor, A very minor issue for approval, updating the Streams product name as alternative product for RHOSAK, RIP :) Thanks!

Thanks @smccarthy-ie. I suggested removing Red Hat from the link.  It's not meant to have it.

**Review by PaulRMellor — APPROVED** [review]

**Comment by smccarthy-ie** [comment]
Cool, thanks Paul :)

**Pull request merged**